lib/internal/cluster/master.js

```diff
@@ -115,7 +115,7 @@ function createWorkerProcess(id, env) {
 
   for (var i = 0; i < execArgv.length; i++) {
     const match = execArgv[i].match(
-      /^(--inspect|--inspect-(brk|port)|--debug|--debug-(brk|port))(=\d+)?$/
+      /^(--inspect|--inspect-(brk|port)|--debug|--debug-(brk|port))(=.*)?$/
```

Comment:
Thanks. I'm thinking about addding `process._debugHostname` to avoid complex regex, what do you think?

```diff
diff --git a/lib/internal/cluster/master.js b/lib/internal/cluster/master.js
index c7b06b8..2a7379b 100644
--- a/lib/internal/cluster/master.js
+++ b/lib/internal/cluster/master.js
@@ -124,7 +124,7 @@ function createWorkerProcess(id, env) {
         ++debugPortOffset;
       }

-      execArgv[i] = match[1] + '=' + debugPort;
+      execArgv[i] = match[1] + '=' + process._debugHostname + ':' + debugPort;
     }
   }

diff --git a/src/node.cc b/src/node.cc
index c241f73..f091ac7 100644
--- a/src/node.cc
+++ b/src/node.cc
@@ -3361,6 +3361,8 @@ void SetupProcessObject(Environment* env,
                              DebugPortSetter,
                              env->as_external()).FromJust());

+  READONLY_PROPERTY(process, "_debugHostname", OneByteString(env->isolate(), debug_options.host_name().c_str()));
+
   // define various internal methods
   env->SetMethod(process,
                  "_startProfilerIdleNotifier",
```